src/components/CovMap.tsx

```diff
@@ -294,12 +294,13 @@ export const CovMap = () => {
       <Legend />
       <Dialog
         aria-labelledby="simple-dialog-title"
-        open={!datasetFound}
+        open={!datasetFound || true}
```

Comment:
```suggestion
        open={true}
```

Reply:
no need to make it more complex than it is 👍

Reply:
ohh wow guess he didn't save... should be only !datasetFound....   OR true was only for debugging to always show it

Reply:
should be fine now :)